### ISSUE TITLE
Status Selection box shows default statuses only

### ISSUE THREAD

**Issue by RedSpid3r** [fmancardi/mantis-triggerclose#3]
The status selection box shows the default mantis names "feedback, confirmed" etc. Would be much better UX if they displayed the ones set by `$s_status_enum_string`

For example, my `feedback` status is renamed to `Need More Info`

it also does not show any custom statuses

**Comment by fmancardi** [comment]
Please use => https://github.com/fmancardi/mantis-triggerclose/tree/mantisbt_2.x.y, test and provide feedback

**Comment by RedSpid3r** [comment]
yep it is now showing custom status but the "Close As User" option is broken - it only shows 1 option which is a random user, so i have to revert for now

**Comment by fmancardi** [comment]
mmm. It has to display only users that are developers (default access level).
can you provide detailed steps to reproduce?


Il 04/Feb/2018 19:02, "RedSpid3r" <notifications@github.com> ha scritto:

**Comment by RedSpid3r** [comment]
I have 22 users assigned either developer "or above". Maybe it is not detecting custom levels?

**Edit:**
Confirmed it does not see custom levels.
I changed `" WHERE access_level=" . DEVELOPER .` to `" WHERE access_level=" . 40 .` 
on line 35 of `/TriggerClose/pages/config.php` to fix

**Comment by RedSpid3r** [comment]
does not seem to be closing any issues, even though they are older than what i set in the config page

**Comment by fmancardi** [comment]
please provide very detailed steps and data to reproduce

Il 12/Feb/2018 12:52, "RedSpid3r" <notifications@github.com> ha scritto:

**Comment by RedSpid3r** [comment]
created a separate issue for this:
https://github.com/fmancardi/mantis-triggerclose/issues/4
